libs/table/cells/LinkCell.tsx

```diff
@@ -13,8 +13,14 @@ export const linkCell =
   (makeHref: (value: string) => string) =>
   ({ value }: Cell<string>) => {
     return (
-      <Link className="text-sans-semi-md text-default hover:underline" to={makeHref(value)}>
+      <Link
+        className="flex h-full w-full items-center text-sans-semi-md text-default hover:underline"
+        aria-label={value}
```

Comment:
is this to prevent the div from messing with it? in general I would expect links to get their accessible name from the text, which is already there

Reply:
Will fix that – was a fix no longer needed